### ISSUE TITLE
Webui is "started" even when explicitly disabled

### ISSUE THREAD

**Issue by paranoidi** [Flexget/Flexget#2561]
### Expected behaviour:

No log messages about webui

### Actual behaviour:

```
2020-01-08 18:09:58 INFO     web_server                    Web interface available at http://127.0.0.1:3539
```

#### Config:
```
web_server:
  bind: 0.0.0.0
  port: 3539
  web_ui: no
```

Probably just a simple logging issue?

**Comment by gazpachoking** [comment]
I think it's probably the api that's started, and the logging is misleading. Just based on memory though, needs investigation.

**Comment by liiight** [comment]
yeah: https://github.com/Flexget/Flexget/blob/7959f64381346a8f2a7b24e414a42827783f5bb6/flexget/plugins/daemon/web_server.py#L97

We probably need to have a better distinction between API and WebUI, but they both obviously need the web server to run. 
Any phrasing suggestions?

**Comment by gazpachoking** [comment]
I guess 'web server' is fine for the general web server. But then it would be nice to have messages for each app on the web server that was started. e.g.
```
2020-01-08 18:09:58 INFO     web_server                    Web server started at http://127.0.0.1:3539
2020-01-08 18:09:58 INFO     web_server                    API available at http://127.0.0.1:3539/api
2020-01-08 18:09:58 INFO     web_server                    Web-ui available at http://127.0.0.1:3539/
```

**Comment by kristenmills** [comment]
We should just change the messaging on the web server one to be about the API. There is a separate message that logs when webui is registered
https://github.com/Flexget/Flexget/blob/7959f64381346a8f2a7b24e414a42827783f5bb6/flexget/plugins/daemon/web_server.py#L111

**Comment by kristenmills** [comment]
i didn't see @gazpachoking comment when i posted this...i like that

**Comment by gazpachoking** [comment]
This is what it looks like now if you start the web server with all possible apps running:
```
2020-01-11 11:25:08 INFO     web_server                    Web server started at http://192.168.0.143:5050
2020-01-11 11:25:08 INFO     web_server                    API available at http://192.168.0.143:5050/api
2020-01-11 11:25:08 INFO     web_server                    WebUI (v1) available at http://192.168.0.143:5050/v1/
2020-01-11 11:25:08 INFO     web_server                    WebUI (v2) available at http://192.168.0.143:5050/
```
Not sure if other log messages need to be tweaked as well, or maybe levels moved down to debug or something.
